### DISCUSSION TITLE
powerpc64/ppc64 version

### DISCUSSION THREAD

**Discussion by iamhumanipromise** [kth5/archpower#49] [Ideas]
Hello! A version compiled for the ppc64 architecture would be great if possible! I believe this would work for the Apple PowerMac G5 as well as the POWER4 systems. (Anyone please correct me if I am wrong, I don't know the earliest generation specifics)

**Comment by DarjanKrijan** [comment]
The powerpc image has both powerpc and powerpc64, selectable in grub.

**Comment by kth5** [comment]
Indeed, it might not be immediately obvious but the `powerpc` ISOs have the option to boot a `ppc64` kernel that will support G5 and potentially other compatible 64bit POWER ISA compatible platforms, albeit with a 32bit userland.

**Comment by kth5** [comment]
I got a G5 coming in tomorrow, we shall see that I get this tested more. Just wish these things would be available in a small form factor like a mini. :')

**Reply by iamhumanipromise** [reply]
SFF would be a lot nicer than a G5! But glad to hear you have one arriving! 

The POWER4 - POWER8 machines would be awesome also but are a huge rack mounted affair... unless you can manage to find one of the workstations!

Do you have a fundraiser page for hardware?
